services/blockchain-coordinator/shared/scheduler.js

```diff
@@ -94,7 +93,7 @@ const initIndexingScheduler = async () => {
 	Signals.get('newBlock').add(newBlockListener);
 
 	// Retrieve enabled modules from connector
-	setRegisteredModules(await getEnabledModules());
+	registeredLiskModules = await getRegisteredModules();
```

Comment:
the `setRegisteredModules` logic is removed? we dont need to set after getting them?